streampark-flink/streampark-flink-connector/streampark-flink-connector-base/src/main/scala/org/apache/streampark/flink/connector/failover/SinkRequest.scala

```diff
@@ -30,21 +30,21 @@ case class SinkRequest(records: util.List[String], var attemptCounter: Int = 0)
   def size: Int = records.size()
 
   private[this] lazy val TABLE_REGEXP =
-    Pattern.compile("(insert\\s+into|update|delete)\\s+(.*?)(\\(|\\s+)", Pattern.CASE_INSENSITIVE)
+    Pattern.compile("^\\s*(insert\\s+into|update|delete)\\s+(.*?)(\\(|\\s+)", Pattern.CASE_INSENSITIVE)
 
   private[this] lazy val INSERT_REGEXP =
-    Pattern.compile("^(.*)\\s+(values|value)(.*)", Pattern.CASE_INSENSITIVE)
+    Pattern.compile("^\\s*(insert\\s+into)\\s+(`?([a-zA-Z]\\w*)`?.)?`?([a-zA-Z]\\w*)`?((\\s*\\((\\s*`?[a-zA-Z]\\w*`?\\s*)*\\))|(\\s+))\\s*(value|values)\\s*(\\(.*\\))\\s*;?", Pattern.CASE_INSENSITIVE)
```

Comment:
The regex expression here is too complicated, can you make it simpler ?

Reply:
Thank you for your reminder, I now use another method to fix this bug

Reply:
I found the simplest solution, which only needs to add one character to the original code. The commit is "
refactor: amazing solution"